packages/debug-nodejs/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@theia/debug-nodejs",
+  "version": "0.3.13",
+  "description": "Theia - NodeJS Debug Extension",
+  "dependencies": {
+    "@theia/debug": "^0.3.13",
+    "vscode-debugprotocol": "^1.26.0"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "theiaExtensions": [
+    {
+      "backend": "lib/node/debug-nodejs-backend-module"
+    }
+  ],
+  "keywords": [
+    "theia-extension, debug, nodejs"
+  ],
+  "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/theia-ide/theia.git"
+  },
+  "bugs": {
+    "url": "https://github.com/theia-ide/theia/issues"
+  },
+  "homepage": "https://github.com/theia-ide/theia",
+  "files": [
+    "lib",
+    "src"
+  ],
+  "scripts": {
+    "prepare": "yarn run clean && yarn run build",
+    "clean": "theiaext clean",
+    "build": "concurrently -n download,build -c red,blue \"gulp download_nodejs_debug_adapter\" \"theiaext build\"",
+    "watch": "theiaext watch",
+    "test": "theiaext test",
+    "docs": "theiaext docs"
+  },
+  "devDependencies": {
+    "@theia/ext-scripts": "^0.3.13",
+    "gulp": "^3.9.1",
```

Comment:
Please, if there is nothing else used from `gulp`,  could you convert it to plain nodejs like in `@theia/java` it's huge library to just download an archive in the end. Thanks!

Reply:
In @tolusha 's defence, I'm surprised Theia doesn't use a task runner like gulp already. IMO, the npm scripts and `theia-cli` are alien and unwieldy. Perhaps a discussion for another time, however :)

Reply:
We can bring the matter for next week's dev meeting?
I would be interested to hear about it :)

Reply:
It is not about Theia, but about downstream projects, they will have to install gulp just to download binaries.

Reply:
Using npm scripts as build tools is a common practice in JS world: https://www.keithcirkel.co.uk/how-to-use-npm-as-a-build-tool/
As well as abstracting common scripts via packages: https://blog.kentcdodds.com/automation-without-config-412ab5e47229

I don't really see a need to pull more dev dependencies and write additional js files for them.